dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.mail-2.0.feature

```diff
@@ -6,11 +6,9 @@ Subsystem-Version: 2.0
 IBM-Process-Types: client, \
  server
 -features=\
-  io.openliberty.jakarta.activation-2.0, \
- com.ibm.websphere.appserver.eeCompatible-9.0
```

Comment:
Why did you remove eeCompatible-9.0?

Reply:
Trying to figure out why this feature is looking for the Jakarta mail implementation packages. It should not need them. Removed the eeCompatible-9.0 to see If that is the reason.

Reply:
eeCompatible-9.0 isn't the cause of that, it doesn't bring in any code itself. Please put it back.